packages/squiggle-lang/src/fr/plot.ts

```diff
@@ -96,6 +97,84 @@ const _assertYScaleNotDateScale = (yScale: Scale | null) => {
   }
 };
 
+const numericFnDef = () => {
```

Comment:
Note that I did most of these, with tests, in this PR, so these defs don't need to be reviewed closely.

https://github.com/quantified-uncertainty/squiggle/pull/2676